plugin/src/leiningen/figwheel.clj

```diff
@@ -396,22 +410,34 @@
    (fuzzy-config-from-project project)
    {:no-start-option true}))
 
+(defn add-checkouts [fuzzy-config project]
+  (let [checkout-paths (checkout-source-paths project)]
+    (update-in
+      fuzzy-config
+      [:cljsbuild :builds]
+      (fn [builds]
+        (map (fn [build]
+               (update build :source-paths concat checkout-paths))
+             builds)))))
+
 (defn build-once [project build-ids]
   (when-not (report-if-bad-build-ids project build-ids)
     (run-build-once
      project
      (fuzzy-config-from-project project)
      (source-paths-for-classpath
-      (normalize-data project build-ids))
+      (normalize-data project build-ids)
+      project)
      (vec build-ids))))
 
 (defn figwheel-main [project build-ids]
   (when-not (report-if-bad-build-ids project build-ids)
     (run-figwheel
      project
-     (fuzzy-config-from-project project)
+     (-> project fuzzy-config-from-project (add-checkouts project))
      (source-paths-for-classpath
-      (normalize-data project build-ids))
+      (normalize-data project build-ids)
+      project)
```

Comment:
Or alternatively you can just distinct-concat them onto the classpaths on line 438